algolia/internal/gen/options.go

```diff
@@ -155,6 +155,7 @@ var options = []Option{
 	{"query", Other, "", ""},
 	{"autoGenerateObjectIDIfNotExist", Other, false, ""},
 	{"clearExistingRules", Other, false, ""},
+	{"clearExistingSynonyms", Other, false, ""},
 	{"replaceExistingSynonyms", Other, false, ""},
```

Comment:
What happens if both option have contradictory information?

Reply:
it will take only use replace if clear isn't set https://github.com/algolia/algoliasearch-client-go/pull/634/files#diff-25d7c0cffb7526045febbaea8c194a47dac529e0e14f918ab074774ed8e77554R50-R54